core/src/main/java/com/microsoft/applicationinsights/LowLevelClient.java

```diff
@@ -0,0 +1,119 @@
+package com.microsoft.applicationinsights;
+
+import com.azure.core.http.*;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.util.tracing.Tracer;
+import com.azure.monitor.opentelemetry.exporter.implementation.models.TelemetryItem;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.applicationinsights.internal.authentication.AadAuthentication;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Flux;
+import reactor.util.context.Context;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.ByteBuffer;
+import java.util.Iterator;
+import java.util.List;
+
+// TODO performance testing
+class LowLevelClient {
```

Comment:
Can we rename it to ApplicationInisightsClient?

Reply:
renamed to TelemetryChannel